eth-contracts/test/governance.test.js

```diff
@@ -1737,6 +1749,18 @@ contract('Governance.sol', async (accounts) => {
         "Only callable by self"
       )
       
+      // should revert if attempting to set voting quorum to zero 
+      await _lib.assertRevert(
+        governance.guardianExecuteTransaction(
+          governanceKey,
+          callValue0,
+          'setVotingQuorum(uint256)',
+          _lib.abiEncode(['uint256'], [0]),
+          { from: guardianAddress }
+        ),
+        "Governance::guardianExecuteTransaction: Transaction failed."
+      )
```

Comment:
this should call `setVotingQuorumPercent` now (and change comment accordingly)
also can you add corresponding test for quorumPercent > 100

Reply:
@SidSethi added